Python/cp/house_robber.py

```diff
@@ -0,0 +1,26 @@
+class Solution(object):
+    def rob(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        if nums ==[] or len(nums) == 0:
+            return 0
+        elif len(nums) == 1: 
+            return nums[0]
+        
+
+        runningTotal = [-1,-1]
+        runningTotal[0] = nums[0]
+        runningTotal[1] = max(nums[0], nums[1])
+
+        for i in range(2, len(nums)):
```

Comment:
Change i to index

Reply:
@SKAUL05 
changed i to index. Please check the [new commit](https://github.com/TesseractCoding/NeoAlgo/pull/1315/commits/7040544f067acbe84d152e736360f4a829ed7edf)